.github/workflows/build.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           # - 9.6.1
           - 9.6.2
           - 9.8.1
+          - 9.10.1
         include:
           - os: macos-latest
```

Comment:
This also needs to be changed to `macos-12`, as later versions of the runner image don't include `ghc` / `ghcup` anymore.